app.js

```diff
@@ -58,7 +52,68 @@ var itemObjects = [
 ];
 
 var itemsObjectsWorking;
-var itemsChosen = 0;
+var itemsChosen = window.localStorage.itemsChosen || 0;
+var itemsDisplayed = window.localStorage.itemsDisplayed || 0;
+
+function onLoadValues(){
+  updateItemObjects();
+  if(itemsChosen > 24){
+    var surveyButtonLocation = document.getElementById('startSurvey');
+    surveyButtonLocation.setAttribute('style', 'display:none');
+    buildAPrettyChart();
+  }
+  else if(itemsDisplayed > 0){
+    var surveyButtonLocation = document.getElementById('startSurvey');
+    surveyButtonLocation.setAttribute('style', 'display:none');
+    displayPreviousObjects();
+  }
+}
+
+function updateItemObjects(){
+  if(window.localStorage.itemObjectSavedVallues){
+    var storedObjects = JSON.parse(window.localStorage.itemObjectSavedVallues);
+    itemObjects.map(function(item){
+      item.numberOfTimesShown = storedObjects[item.itemName].numberOfTimesShown;
+      item.usedInLastItemSet = storedObjects[item.itemName].usedInLastItemSet;
+      item.numberOfClicks = storedObjects[item.itemName].numberOfClicks;
+    });
+  }
+}
+
+function displayPreviousObjects(){
+  var locations = ['surveyFirstItem','surveySecondItem','surveyThirdItem'];
+  var lastDisplayed = itemObjects.filter(function(item){
+    if(item.usedInLastItemSet === true){
+      return true;
+    }
+  });
+  console.log(lastDisplayed);
+  lastDisplayed.map(function(item,index){
+    item.buildSurveyItem(locations[index]);
+    item.numberOfTimesShown--; //decrement here because we've already shown once and don't need to count this as a second time
+  });
+}
+
+function writeToStorage(){
+  window.localStorage.itemsChosen = itemsChosen;
+  window.localStorage.itemsDisplayed = itemsDisplayed;
+  window.localStorage.itemObjectSavedVallues = prepForStorage();
+}
+
+function prepForStorage(){
+  var obj = {};
+  for(var i = 0; i < itemObjects.length; i++){
+    obj[itemObjects[i].itemName] = prepData(i);
+  }
+  return JSON.stringify(obj);
+}
+
+function prepData(index){
+  var numberOfTimesShown = itemObjects[index].numberOfTimesShown;
```

Comment:
Once again, good instinct to pull out a function here instead of putting the code directly in `prepForStorage`.

I'd probably pass `itemObjects[index]` into this function directly, since you already need a reference to it where this function is called.